tests/test_runner/test_parallel.py

```diff
@@ -44,6 +55,17 @@ def dummy_test(self):
 
 class RemoteTestResultTest(SimpleTestCase):
 
+    def test_pickle_errors_detection(self):
+        picklable_error = RuntimeError('This is fine')
+        not_unpicklable_error = ExceptionThatFailsUnpickling('one arg', 'another arg')
+
+        result = RemoteTestResult()
+        result._confirm_picklable(picklable_error)
+
+        # __init__() missing 1 required positional argument (different on PY2)
+        with self.assertRaisesMessage(TypeError, 'argument'):
```

Comment:
We could do that. I had planned to simply fill in the Python 3 message in a few months when Python 2 support is dropped in master.